plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -2254,6 +2270,46 @@ protected void configureVideoCard(VirtualMachineMO vmMo, VirtualMachineTO vmSpec
         }
     }
 
+    /**
+     * Sets GPU card details to the one provided in detail vgpu.type (if provided) on {@code vmConfigSpec}.
+     * @param vmMo
```

Comment:
@nitin-maharana I think that these @param without much explanations can be removed.